Survey/Survey/Sources/Navigation/AppScreenType.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  AppScreen.swift
+//  Survey
+//
+//  Created by Taher on 2/11/22.
+//
+
+enum AppScreenType {
+    case entry
```

Comment:
Please add a line break here

Reply:
update in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/41/commits/f10d87bdfe1267e19fe24bb94768bd66860444a7)